pages/common/now.md

```diff
@@ -0,0 +1,27 @@
+# now
+
+> Cloud platform for serverless deployment.
+
+- Deploy current directory:
+
+`now`
+
+- List of deployments:
+
+`now list`
+
+- Display information related to a deployment:
+
+`now inspect {{deployment_url}}`
+
+- Remove a deployment:
+
+`now remove {{deployment_id}}`
+
+- Log in into account or create a new one:
```

Comment:
Perhaps this will read better?

```suggestion
- Log in into an account or create a new one:
```

Also, I assume this is the first thing a user will probably want to do. Could we move this to the top of the page please? tldr-pages usually present commands in the order that they are likely to be used.

Reply:
I thought write about the login command as a hint to way to change the account, to a new one or to existing.

If not authorized user tries to deploy by command now it will start the login process first.
<img width="610" alt="image" src="https://user-images.githubusercontent.com/12317269/51217643-eeb23f00-1939-11e9-81a9-b0071061cead.png">

Reply:
I see. If it prompts for a login automatically, then is this command commonly used?

Reply:
You mean is `now` commonly used command? Yes, after enter `now` occur fully automated deployment with auto-copy generated url to the clipboard and it is the main service thing.

Reply:
Ah, I mean the `now login` command.